br/pkg/task/restore_raw.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/pingcap/errors"
 	"github.com/pingcap/log"
+	kvconfig "github.com/pingcap/tidb/br/pkg/config"
```

Comment:
Why rename it to `kvconfig`? It seems belongs to BR. Perhaps use `pconfig` as `restore.go` did.

Reply:
I think all logic belong to TiKV could use this name